requirements.txt

```diff
@@ -1,6 +1,7 @@
-coveralls>=1.7.0
 six
 testcontainers
+coveralls>=1.7.0
```

Comment:
Something happened here. Why was coveralls put back in. We don't use it. Also please change requests to >=

Reply:
Hmm, I just moved it around: 

```
-coveralls>=1.7.0
 six
 testcontainers
+coveralls>=1.7.0
+requests==2.21.0
```
I feel like @rabitt added it, but am unsure!

Reply:
Ah. I thought I removed it. Guess I didn't... I'll do that in another pr I suppose.